pytensor/tensor/basic.py

```diff
@@ -678,10 +678,9 @@ def make_node(self, t):
             self, [t], [ps.get_scalar_type(dtype=t.type.dtype).make_variable()]
         )
 
-    def perform(self, node, inp, out_):
-        (s,) = inp
-        (out,) = out_
-        out[0] = s.flatten()[0]
+    def perform(self, node, inputs, output_storage):
+        # not using .item() because that returns a Python scalar, not a numpy scalar
+        output_storage[0][0] = inputs[0][()]
```

Comment:
That `[()]` syntax is really ugly

Reply:
You can also use `None` I believe.

Reply:
The idea is you're indexing an array without elements so you don't index anything. Anyway surprised it bothers you too much